app/assets/stylesheets/foundation-portfolio-theme.scss

```diff
@@ -22,7 +22,8 @@ body {
 }
 
 .button-group a:hover {
-  color:rgba(0,0,0,0.4);
+  color:rgba(255,255,255,1);
```

Comment:
Commas in function arguments should be followed by a single space<br>Colon after property should be followed by one space